hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/SpaceQuotaHelperForTests.java

```diff
@@ -271,6 +271,17 @@ void setQuotaLimit(final TableName tn, SpaceViolationPolicy policy, long sizeInM
     LOG.debug("Quota limit set for table = {}, limit = {}", tn, sizeLimit);
   }
 
+  /**
+   * Sets the given quota (policy & limit) on the passed namespace.
+   */
+  void setQuotaLimitNamespace(final String namespace, SpaceViolationPolicy policy, long sizeInMBs)
```

Comment:
Can you please fix `setQuotaLimit()` and `removeQuotaFromtable` to be consistent with these new methods?

e.g. `setQuotaLimitTable()` and `removeQuotaFromTable()`